schemas/slashfeed.json

```diff
@@ -1,13 +1,23 @@
 {
-  "image": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAyNHB4IiBoZWlnaHQ9IjEwMjRweCIgdmlld0JveD0iMCAwIDEwMjQgMTAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgPGNpcmNsZSBjeD0iNTEyIiBjeT0iNTEyIiByPSI1MTIiIHN0eWxlPSJmaWxsOiMwM2NhOWIiLz4KICAgPHBhdGggZD0iTTczMi44MSAyNzVjLTEuNjctLjczLTE3My41MS0yNC42LTM0My4zOSA4Ni44OEMyODQgNDMxLjIyIDI3MCA1MzIuNjggMjc0LjgxIDYwMC4zNmMyNDcuMDgtMjcuODkgNDUyLjQxLTMxNy4yOSA0NTgtMzI1LjM2ek0yOTEuNTcgNjc1LjUzYzIxLjI0IDM4LjE0IDE4MS43NyAxMjQuNiAzMjMuODcgNS4zNVM3NDUuNjIgMzQ0LjY2IDczMi44MSAyNzVjLTQuNDcgMTAuMTEtMTU5LjYyIDM1Ni44OS00NDEuMjQgNDAwLjUxIiBzdHlsZT0iZmlsbDojZmZmIi8+Cjwvc3ZnPgo=",
-  "name":"Bitfinex",
-  "feed_type": "exchange_feed",
+  "name": "Bitfinex",
+  "description": "Bitfinex account feed",
+  "type": "account_feed",
   "version": "0.0.1",
-  "wallets" : [
+  "icons": {
+    "48": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAyNHB4IiBoZWlnaHQ9IjEwMjRweCIgdmlld0JveD0iMCAwIDEwMjQgMTAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgPGNpcmNsZSBjeD0iNTEyIiBjeT0iNTEyIiByPSI1MTIiIHN0eWxlPSJmaWxsOiMwM2NhOWIiLz4KICAgPHBhdGggZD0iTTczMi44MSAyNzVjLTEuNjctLjczLTE3My41MS0yNC42LTM0My4zOSA4Ni44OEMyODQgNDMxLjIyIDI3MCA1MzIuNjggMjc0LjgxIDYwMC4zNmMyNDcuMDgtMjcuODkgNDUyLjQxLTMxNy4yOSA0NTgtMzI1LjM2ek0yOTEuNTcgNjc1LjUzYzIxLjI0IDM4LjE0IDE4MS43NyAxMjQuNiAzMjMuODcgNS4zNVM3NDUuNjIgMzQ0LjY2IDczMi44MSAyNzVjLTQuNDcgMTAuMTEtMTU5LjYyIDM1Ni44OS00NDEuMjQgNDAwLjUxIiBzdHlsZT0iZmlsbDojZmZmIi8+Cjwvc3ZnPgo="
+  },
+  "fields": [
     {
-      "wallet_name" : "Bitcoin",
-      "currency" : "BTC",
-      "unit" : "btc"
+      "name": "Bitcoin",
+      "description": "Bitcoin balance",
+      "main": "/bitcoin/",
+      "type": "number"
```

Comment:
Please note that @rbndg wanted types to be something else, see the comment in Asana:
https://app.asana.com/0/0/1203662225268958/1201596901781209/f

Reply:
Sure, I just had this in place prior to latest update on Asana and it did not appear to me that we decided on something there. I will change it here as soon as decision will be made

Reply:
Maybe add an additional attribute, called ``` "unit":"satoshi ``` so they add the data as Sats

Reply:
@rbndg I've abstained from adding `label` as you have suggested in DMs as it seems to me exactly what `description` is. Please let me know if I am mistaken here